node_name: "mission_executor_node"
node_rate: 10


maximum_expected_action_time_s:
  communicate: 5
  takeoff: 10
  land: 10
  drop_bouya: 10
  travel_to: 120
  track: 120
  search: 180
  idle: 99999


interface_name:
  takeoff: "/high_level_controller/service/request_take_off"
  landing: "/high_level_controller/service/request_landing"


enable_controller_service_names:
  attitude_controller: "/attitude_controller/service/enable_controller"


locations_ned:
  helipad: [0, 0, 0]

